packages/addon/src/menu.ts

```diff
@@ -135,4 +167,8 @@ export function init() {
     secondaryTitle: browser.i18n.getMessage('thunderbirdPro'),
     tooltip: '',
   });
+
+  // Technically this is an async function.
+  // But we do not need to wait for it synchronously.
+  getLoginState();
```

Comment:
Hamburger initial login stage, part 2.

As part of initialization, we call our new login-state-checker.